README.md

```diff
@@ -99,9 +99,11 @@ const createComponent = createComponentFactory({
   providers: [],
   declarations: [],
   entryComponents: [],
-  componentProviders: [], // Override the component's providers 
+  componentProviders: [], // Override the component's providers
```

Comment:
We need to be consistent. Let's change it to `componentViewProvidersMocks`, please.